vispy/color/transfer_function.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+import vispy.gloo
+
+
+class BaseTransferFunction:
```

Comment:
Since this is a user-facing object that people are supposed to subclass, I think we need to explain some stuff in the docstring:
- how to modify it (change the glsl code, update uniforms)
- what needs to stay
- explain a bit the arguments passed in to `applyTransferFunction`